sdk/include/opentelemetry/sdk/metrics/state/attributes_hashmap.h

```diff
@@ -23,6 +23,15 @@ namespace metrics
 
 using opentelemetry::sdk::common::OrderedAttributeMap;
 
+constexpr uint16_t kAggregationCardinalityLimit = 2000;
+const std::string kAggregationCardinalityLimitOverflowError =
+    "Maximum data points for metric stream exceeded. Entry added to overflow";
+const std::string kAttributesLimitOverflowKey   = "otel.metrics.overflow";
+const std::string kAttributesLimitOverflowValue = "true";
```

Comment:
Per the spec:

"
An overflow attribute set is defined, containing a single attribute otel.metric.overflow having (boolean) value true, which is used to report a synthetic aggregation of the metric events that could not be independently aggregated because of the limit.
"

The overflow value should be a boolean true, not a string "true".